l10n/en/newsletter_form.ftl

```diff
@@ -36,6 +36,7 @@ newsletter-form-thanks = Thanks!
 newsletter-form-leave-this-field-empty = Leave this field empty.
 newsletter-form-yes = Yes
 newsletter-form-no = No
+newsletter-form-label = Newsletter form
```

Comment:
Don't know whether newsletters.ftl is also loaded here, so any of these would be available instead:?

https://github.com/mozilla/bedrock/blob/4b32207c710d3c680d26e1b0ec80c529af5dafe2/l10n/en/mozorg/newsletters.ftl#L8-L11

Reply:
(I was originally considering linking an **id** of a heading instead, but none of the headings clearly state it's _a newsletter_ … so a plain label saying just that seems better.)

Reply:
I think we need something generic that does not imply what type of newsletter it is on the page.